aiogram/filters/base.py

```diff
@@ -37,6 +32,14 @@ async def __call__(self, *args: Any, **kwargs: Any) -> Union[bool, Dict[str, Any
     def update_handler_flags(self, flags: Dict[str, Any]) -> None:
         pass
 
-    def __await__(self):  # type: ignore # pragma: no cover
-        # Is needed only for inspection and this method is never be called
-        return self.__call__
+    def _signature_to_string(self, *args: Any, **kwargs: Any) -> str:
+        items = [repr(arg) for arg in args]
+        items.extend([f"{k}={v!r}" for k, v in kwargs.items()])
+
+        return f"{type(self).__name__}({', '.join(items)})"
+
+    def __str__(self) -> str:
+        return self._signature_to_string()
+
+
+BaseFilter = Filter
```

Comment:
not imported to `__init__`. why it here?

Reply:
Fixed.

But also I'm not sure that the `BaseFilter` alias is needed in the final version.